### DISCUSSION TITLE
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target

### DISCUSSION THREAD

**Discussion by bourribab** [DependencyTrack/dependency-track#1381] [General]
Hello
I have created my keystore with the password.

how can i import it on dependency track apiserver 
Is it via the java_options environment variable ?

- JAVA_OPTIONS= "-Djavax.net.ssl.trustStore=/data/depedency_track/keystore/keystore.jks -Djavax.net.ssl.trustStorePassword=******"

When I use this conf it doesn't work because it can find the class

Any idea PLZ?

**Comment by stevespringett** [comment]
I think your truststore would have to contain all certificates from the default Java truststore, in addition to any certificates from your organization. Configuration of truststore is not specific to Dependency-Track.
